packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js

```diff
@@ -164,15 +127,30 @@ export default class {
 
   onNetworkTypeChange(usePrivateNetwork) {
     if (!usePrivateNetwork) this.model.subnet = null;
+    this.getOrderData();
+  }
+
+  // bug in oui-numeric: on-change event is fired before model is updated
+  onNodeNumberChange() {
+    this.$timeout(() => this.getOrderData());
+  }
+
+  getNodesSpecTranslation() {
+    const prefix = 'pci_database_add_spec_nodes';
+    const range =
+      this.model.plan.nodesCount === this.model.plan.maxNodes ? '' : '_range';
+    const single = this.model.plan.nodesCount === 1 ? '_single' : '';
+    const translateKey = `${prefix}${range}${single}`;
+    return this.$translate.instant(translateKey, {
+      min: this.model.plan.nodesCount,
+      max: this.model.plan.maxNodes,
+    });
```

Comment:
for readability better like this :
```suggestion
    const { nodesCount, maxNodes } =  this.model.plan;
    const prefix = 'pci_database_add_spec_nodes';
    const range =nodesCount === maxNodes ? '' : '_range';
    const single = nodesCount === 1 ? '_single' : '';
    const translateKey = `${prefix}${range}${single}`;
    
    return this.$translate.instant(translateKey, {
      min: nodesCount,
      max: maxNodes,
    });
```

Reply:
Better indeed ! Done here: https://github.com/ovh/manager/pull/6905/commits/15ac9b6b17cc909010729c53281b391ff78b7e59